app/not-found.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from 'next/link';
+
+export default function NotFound() {
+  return (
+    <div>
+      <h1>Not found: 404!</h1>
+      <div>
+        <Link href="/">Go back to Home</Link>
+      </div>
+    </div>
+  );
```

Comment:
not-found, error, and global-error files are added but they're temporary. We'll come back to make these better.